control/jet_vane_model_test.cc

```diff
@@ -0,0 +1,156 @@
+#include "control/jet_vane_model.hh"
+
+#include <cassert>
+
+namespace jet {
+namespace control {
+
+// TODO: Setup gtest
+
+void test_force_monotonic_in_throttle() {
+  //
+  // Setup
+  //
+
+  VaneConfiguration vane_cfg;
+  vane_cfg.T_vane_unit_from_zero = SE3();
+
+  const JetConfiguration jet_cfg;
+  const double commanded_servo_angle_rad = 0.0;
+
+  const JetStatus status_zero({.throttle = 0.0});
+  const JetStatus status_half({.throttle = 0.5});
+  const JetStatus status_full({.throttle = 1.0});
+
+  //
+  // Action
+  //
+
+  const Wrench wrench_zero_throttle = vane_unit_wrench(commanded_servo_angle_rad, status_zero, vane_cfg, jet_cfg);
+  const Wrench wrench_half_throttle = vane_unit_wrench(commanded_servo_angle_rad, status_half, vane_cfg, jet_cfg);
+  const Wrench wrench_full_throttle = vane_unit_wrench(commanded_servo_angle_rad, status_full, vane_cfg, jet_cfg);
+
+  //
+  // Verification
+  //
+
+  const double drag_zero = wrench_zero_throttle.force_N[0];
+  const double drag_half = wrench_half_throttle.force_N[0];
+  const double drag_full = wrench_full_throttle.force_N[0];
+
+  const double lift_zero = wrench_zero_throttle.force_N[1];
+  const double lift_half = wrench_half_throttle.force_N[1];
+  const double lift_full = wrench_full_throttle.force_N[1];
+
+  assert(drag_zero == 0.0);
+  assert(drag_half > 0.0);
+  assert(drag_full > 0.0);
+
+  assert(drag_zero < drag_half);
+  assert(drag_half < drag_full);
+
+  assert(lift_zero == 0.0);
```

Comment:
Add a comment that these will not be 0 if we add nonzero swirl to the model in the future?